src/plugins/notebook/monkeyPatchObjectAPIForNotebooks.js

```diff
@@ -30,16 +31,53 @@ export default function (openmct) {
     };
 }
 
-function resolveConflicts(localMutable, openmct) {
-    const localEntries = JSON.parse(JSON.stringify(localMutable.configuration.entries));
+function resolveConflicts(domainObject, localMutable, openmct) {
+    if (isNotebookType(domainObject)) {
+        return resolveNotebookEntryConflicts(localMutable, openmct);
+    } else if (isAnnotationType(domainObject)) {
+        return resolveNotebookTagConflicts(localMutable, openmct);
+    }
+}
 
-    return openmct.objects.getMutable(localMutable.identifier).then((remoteMutable) => {
-        applyLocalEntries(remoteMutable, localEntries, openmct);
+async function resolveNotebookTagConflicts(localAnnotation, openmct) {
+    const localClonedAnnotation = structuredClone(localAnnotation);
+    const remoteMutable = await openmct.objects.getMutable(localClonedAnnotation.identifier);
 
-        openmct.objects.destroyMutable(remoteMutable);
+    // should only be one annotation per targetID, entryID, and tag; so for sanity, ensure we have the
+    // same targetID, entryID, and tags for this conflict
+    if (!(_.isEqual(remoteMutable.tags, localClonedAnnotation.tags))) {
+        throw new Error('Conflict on annotation\'s tag has different tags than remote');
+    }
 
-        return true;
+    Object.keys(localClonedAnnotation.targets).forEach(localTargetKey => {
+        if (!remoteMutable.targets[localTargetKey]) {
+            throw new Error(`Conflict on annotation's target is missing ${localTargetKey}`);
+        }
+
+        if (remoteMutable.targets[localTargetKey].entryId !== localClonedAnnotation.targets[localTargetKey].entryId) {
```

Comment:
Should this be 

```js
if (remoteMutable.targets[localTargetKey].entryId !== localClonedAnnotation.targets[localClonedAnnotation].entryId) {
```

as its referred to in the error message? We should consider declaring a couple variables here for increased readability.

Reply:
The error message variable wasn't quite right - nice catch!